test/pages/HaveYouUsedMethodOneInPastPageSpec.scala

```diff
@@ -28,4 +31,18 @@ class HaveYouUsedMethodOneInPastPageSpec extends PageBehaviours {
 
     beRemovable[Boolean](HaveYouUsedMethodOneInPastPage)
   }
+  "cleanup" - {
+    "should reset DescribeTheIdenticalGoodsPage" - {
+      "when HaveYouUsedMethodOneInPastPage is changed to No" in {
+        val emptyUserAnswers = UserAnswers("id", DraftId(1))
+
+        val ua = emptyUserAnswers
+          .set(DescribeTheIdenticalGoodsPage, "identicalities")
+          .get
+
+        HaveYouUsedMethodOneInPastPage.cleanup(Some(false), ua) mustBe
+          Success(emptyUserAnswers)
+      }
+    }
+  }
```

Comment:
We want to be testing all scenarios here, namely what happens when the cleanup is called with a None, and Some(true).